assignment-6&7/src/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import {App, BodyComponent} from './App';
+
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { AboutUs, ErrorComponent, TeamMember } from './Components';
+
+const AppRouter = createBrowserRouter([
+    {
+        path: "/",
+        element: <App />,
+        errorElement: <ErrorComponent />,
+        children: [
+            {
+                path: "/",
+                element: <BodyComponent />
+            },
+            {
+                path: "/member/:login",
+                element: <TeamMember />
+            }, 
+            {
+                path: "/about-us",
+                element: <AboutUs />
+            },
+        ]
+    }
+    
+    
+])
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+    <RouterProvider router={AppRouter} />
+);
+
```

Comment:
The index.js inside components folder will export all the components I have created to App.js. Instead of importing the components one by one, which will clutter further. I have exported all the components from a single file.